### PR TITLE
Update Contact Us Page

### PR DESCRIPTION
**Pull request by jason-ng-zq99**

### CODE REVIEW

**Comment by jason-ng-zq99** [comment]
[update contact us page](https://trello.com/c/YtJW6W4W/41-update-contact-us-page)

File: src/pages/Contact/styles/Contact.styled.ts [hunk 1]
```diff
@@ -50,8 +51,8 @@ export const SectionContent = styled.div`
 export const ContactLink = styled.a`
   font-style: normal;
   font-weight: 300;
-  font-size: clamp(1vw, 2vw, 30px);
-  color: inherit;
+  font-size: clamp(1vw, 2vw, 24px);
```
Comment (hui444) [inline src/pages/Contact/styles/Contact.styled.ts:54]: why the min using vw, usually min n max are absolute values; also can this use values from global styles?

**Review by hui444 — APPROVED** [review]
minor issue, do fix; lgtm

File: src/pages/Contact/styles/Contact.styled.ts (hunk 1 above)
Comment (jason-ng-zq99) [reply src/pages/Contact/styles/Contact.styled.ts]: Resolved in 9fd696b

Added padding left to ensure space between each contact link

**Review by rldefa — APPROVED** [review]
lgtm

**Pull request merged**